src/styles/blocks/nav-menu.scss

```diff
@@ -0,0 +1,49 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+.nav__list {
+  @extend %resetMarginPadding ;
+  display: flex;
+  list-style: none;
+  flex-wrap: wrap;
+  align-items: center;
+
+  &-item {
```

Comment:
```suggestion
  &__list-item {
```
Fix all cases

Reply:
I'm not sure on this, please check on my recent deployment